src/backend/joanie/core/serializers/client.py

```diff
@@ -1057,6 +1058,10 @@ def to_internal_value(self, data):
             }
         )
 
+    def get_amount_currency(self, *args, **kwargs) -> str:
```

Comment:
Following the same reflexion of @lunika, I would update also the method [such as](https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield) :
```suggestion
    def get_currency(self, obj) -> str:
        ...
```